tests/fixture/testnet/config.go

```diff
@@ -372,6 +376,21 @@ func NewTestGenesis(
 		)
 	}
 
+	// Define C-Chain genesis
+	cChainGenesis := &core.Genesis{
+		Config: &params.ChainConfig{
+			ChainID: big.NewInt(43112), // Arbitrary chain ID is arbitrary
+		},
+		Difficulty: big.NewInt(0),     // Difficulty is a mandatory field
+		GasLimit:   uint64(0x5f5e100), // Arbitrary gas limit is arbitrary
+		Alloc:      cChainBalances,
+	}
```

Comment:
I'm no EVM expert but someone who is should have a look and make sure we're not omitting some important field(s)